datamade/how-to
# Add recommendation of adoption, documentation and template for vanilla JavaScript development stack

## Overview

See title.

Handles #195.

## Testing Instructions

* Review the new document for soundness and completeness.
* Confirm links resolve.
* Follow the instructions in `docker/templates/` to create a vanilla JS app from the new cookiecutter. Confirm you can follow the instructions in the generated README to run your new app.
